lib/Dialect/TensorExt/IR/TensorExtOps.td

```diff
@@ -23,6 +23,10 @@ def TensorExt_RotateOp : TensorExt_Op<"rotate", [Pure, AllTypesMatch<["tensor",
     This corresponds to the `rotate` operation in arithmetic FHE schemes like
     BGV.
 
+    This op will be adjusted to support rotations of higher dimensional tensors
+    by requiring a vector of integers for each dimension to rotate. The lowering
+    should also take into account the packing of the tensor.
```

Comment:
Might want to clarify here something like this:

```
This operation's current behavior allows rotating multi-dimensional tensors by rotating along the tensor's only non-unit dimension. This assumes the tensor is packed along the non-unit dimension. 
```

I think you can put the "future work" into a comment to avoid confusion for the reader.
```
// In the future, the op will be adjusted to support rotations of general multi-dimensional tensors with a vector of rotation indices for each dimension. The lowering will implement the correct operations to rotate the tensor along the indices given its packing.
```

Reply:
For the first half of your comment,
> This operation's current behavior allows rotating multi-dimensional tensors by rotating along the tensor's only non-unit dimension. This assumes the tensor is packed along the non-unit dimension. 

I thought that it currently doesn't allow rotating multi-dimensional tensors (where the tensor has only a single non-unit dimension - unless you or someone else fixed that?

Reply:
Oh, I mean your PR updates it!